src/TestFSharpInterfaces/TestFSharpInterfaces.fsproj

```diff
@@ -0,0 +1,77 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
`TestFsharp` contains grain implementations and thus references `TestGrainInterfaces`. Adding the F# interface definition in the project would result in a circular dependency between `TestFsharp` and `TestGrainInterfaces`.
Adding `TestFSharpInterfaces` avoids that and allows testing the target scenario (deriving a grain interface from an F# uinterface)